src/python/visualize_matching_result.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import cv2
+import argparse
+from pathlib import Path
+import protos_io
+
+
+def createHiddenMatchImage(width, height):
+    image = np.zeros((height, width, 3), dtype=np.uint8)
+    text_left_corner = (int(width / 3), int(height / 2))
+    image = cv2.putText(
+        image,
+        "No match found",
+        text_left_corner,
+        cv2.FONT_HERSHEY_SIMPLEX,
+        1,
+        (255, 255, 255),
+        2,  # thickness
```

Comment:
nit: It's probably a bit better to actually create a named variable for this and other magic numbers in this function but it's not too important.

Reply:
Added the names of the parameters as comments.